client/src/app/primitives/icon/font/app-codes.css

```diff
@@ -1,13 +1,14 @@
 
+.app-icon-align-bottom-tool:before { content: '\e800'; } /* '' */
 .app-icon-align-left-tool:before { content: '\e801'; } /* '' */
 .app-icon-align-center-tool:before { content: '\e802'; } /* '' */
 .app-icon-align-right-tool:before { content: '\e803'; } /* '' */
 .app-icon-align-top-tool:before { content: '\e804'; } /* '' */
-.app-icon-align-middle-tool:before { content: '\e808'; } /* '' */
-.app-icon-align-bottom-tool:before { content: '\e800'; } /* '' */
 .app-icon-deploy:before { content: '\e805'; } /* '' */
 .app-icon-distribute-horizontal-tool:before { content: '\e806'; } /* '' */
 .app-icon-distribute-vertical-tool:before { content: '\e807'; } /* '' */
+.app-icon-align-middle-tool:before { content: '\e808'; } /* '' */
+.app-icon-play:before { content: '\e809'; } /* '' */
```

Comment:
Instead of adding the `play` icon to the  font, we could decide to add it as an svg component, cf. https://github.com/camunda/camunda-modeler/pull/1583

Reply:
But should be fine to have it inside the font.

Reply:
What would be the benefit of that?

Reply:
Less work to do in the future. The icon font is already a technical debt.

Reply:
Yes. We should start to use SVG icons directly. We have the infrastructure for that, now.

Reply:
For reference, you can use [this code](https://github.com/camunda/camunda-modeler/blob/4e7e28d1ff65ad0be0b71d98a45f664a99d905ff/client/src/plugins/camunda-plugin/deployment-tool/DeploymentConfigModal.js#L20)

Reply:
Updated to use the [play icon svg](https://github.com/camunda/camunda-modeler/pull/1577/files#diff-dd44b266f55cb8f202b75a031e5e852c) instead.